package.json

```diff
@@ -24,7 +24,7 @@
   },
   "devDependencies": {
     "ava": "^0.19.1",
-    "sinon": "^2.2.0",
+    "sinon": "^9.0.2",
```

Comment:
needed for `sinon.fake()`

Reply:
Actually, I see that sinon v9 dropped support for node v8. Should we also do the same, or downgrade to sinon v8?

Reply:
downgraded for the time being